code/game/objects/structures/watercloset.dm

```diff
@@ -5,6 +5,8 @@
 	icon_state = "toilet00"
 	density = FALSE
 	anchored = TRUE
+	can_buckle = 1
```

Comment:
1 --> TRUE

Reply:
I tested this locally. Can_buckle has to equal 1.

Reply:
Reference > https://github.com/Citadel-Station-13/Citadel-Station-13/blob/f2fc50119d33b36d1daccf33d2fc103f44c95ab9/code/game/objects/structures/beds_chairs/chair.dm